### ISSUE TITLE
"Error: Invalid Cast" when performing any download with version 6.5.1, 6.5.2, and 6.5.3

### ISSUE THREAD

**Issue by ceasley** [maurice-daly/DriverAutomationTool#202]
======== Detecting Deployment Platform ========
Deployment Platform: SCCM - Validating ConfigMgr Server Details
======== Querying Model List(s) ========	
Info: Downloading Dell product list	
Info: Downloading Dell product cabinet file from https://downloads.dell.com/catalog/DriverPackCatalog.cab	
======== Errors(s) Occurred ========	
Error: Specified cast is not valid.

This error appears to prevent BIOS or driver downloads from working.

**Comment by dsolodow** [comment]
Are you still seeing this? I nuked the catalog files from the Temp directory and it downloaded & extracted the Dell catalog without issue. Using 6.5.3

**Comment by ceasley** [comment]
Yes, I actually only see this when there are no files in Temp.  If I put
the .cab and .xml back, it seems to work.  It's as if it is unable to
download the .cab and/or xml from Dell.
